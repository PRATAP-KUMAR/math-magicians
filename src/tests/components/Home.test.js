import { MemoryRouter } from 'react-router-dom';
import { render, screen } from '@testing-library/react';

import Home from '../../components/Home';

describe('Home Component Test', () => {
  test('test for Home compo', () => {
    render(<MemoryRouter><Home /></MemoryRouter>);
    expect(screen.getByText('Welcome to our page!')).toMatchSnapshot();
  });
});
