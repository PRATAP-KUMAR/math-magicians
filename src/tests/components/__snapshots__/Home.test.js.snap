// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Test test for Home compo 1`] = `
<h2>
  Welcome to our page!
</h2>
`;
